app/Platform/Components/GameCard.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Platform\Components;
+
+use App\Community\Enums\AwardType;
+use App\Community\Enums\ClaimStatus;
+use App\Community\Models\AchievementSetClaim;
+use App\Platform\Models\Game as GameModel;
```

Comment:
Is the alias needed? Usually resources in model form do not get a type suffix, any other class for the same resource would (e.g. `GameResource`, `GameFactory`, ... )